var/spack/repos/builtin/packages/yaml-cpp/package.py

```diff
@@ -39,7 +40,7 @@ class YamlCpp(CMakePackage):
     variant('pic',   default=True,
             description='Build with position independent code')
 
-    depends_on('boost', when='@:0.5.3')
```

Comment:
you need to add
```
depends_on('boost', when='@0.5.4:')
```

Reply:
No, sorry for not adding that in the description. There is no boost dependency in the `0.6.x` series anymore. (https://github.com/jbeder/yaml-cpp/releases) on the other hand `c++11` is now required, but I don't know how to specify this

Reply:
ah, i see.

Reply:
> c++11 is now required, but I don't know how to specify this

conflicts for compiler versions below some, i.e. gcc:4.8 or alike.